react-ui/src/utils/useSwrContest.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Get data from current contest, or override with specific contest
+ */
+
+import useContestId from './useContestId';
+import useSwrAuth from './useSwrAuth';
+
+// revalidateIfStale = false
+const useSwrContest = (overrideId, revalidateOnMount = false) => {
+  const contestId = overrideId || useContestId();
+  const apiPath = `/contests/${contestId}`;
+
+  const { data = {}, error: e, ...rest } = useSwrAuth(apiPath, { ...revalidateOnMount });
```

Comment:
What does it mean to use the spread operator on a `bool`?

Reply:
It doesn't mean anything. I think I had converted to `options` at once point, then sloppily reverted back. Fixed now.